pkgs/development/python-modules/crewai/default.nix

```diff
@@ -0,0 +1,111 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  gitUpdater,
+
+  # build-system
+  hatchling,
+
+  # dependencies
+  pydantic,
+  langchain,
+  openai,
+  opentelemetry-api,
+  opentelemetry-sdk,
+  opentelemetry-exporter-otlp-proto-http,
+  instructor,
+  regex,
+  crewai-tools,
+  click,
+  python-dotenv,
+  appdirs,
+  jsonref,
+  json-repair,
+  auth0-python,
+  litellm,
+  pyvis,
+  uv,
+  tomli-w,
+  tomli,
+  chromadb,
+
+  # optionals
+  agentops,
+  mem0ai,
+}:
+
+buildPythonPackage rec {
+  pname = "crewai";
+  version = "0.80.0";
+  pyproject = true;
+  disabled = pythonOlder "3.10";
+  src = fetchFromGitHub {
+    owner = "crewAIInc";
+    repo = "crewAI";
+    rev = "refs/tags/${version}";
+    hash = "sha256-MVWNjV2S2l96tEeaH+3sKftUVRDSkx0dLpfMbWsRRdc=";
+  };
+  passthru.updateScript = gitUpdater { };
```

Comment:
This file is hard to follow, can you please group attributes related to testing together and move it just before the meta ? Also, add a few blank lines to make them easier to read.